de.dlr.sc.overtarget.language.ui/src/de/dlr/sc/overtarget/language/ui/quickfix/OvertargetQuickfixProvider.xtend

```diff
@@ -18,6 +22,17 @@ import org.eclipse.xtext.ui.editor.quickfix.DefaultQuickfixProvider
  */
 class OvertargetQuickfixProvider extends DefaultQuickfixProvider {
 
+	@Fix(OvertargetValidator.DEPRECATED_WS_STATEMENT)
+	def fixDeprecatedWsStatement(Issue issue, IssueResolutionAcceptor acceptor) {
+		acceptor.accept(issue, 'Fix Working System', 'Replace with Correct Windowing System.', 'upcase.png') [
```

Comment:
This Quickfix is still the example one. Please update it to change from WorkingSystem statement to WindowingSystem Statement.